iteration/iteration_test.go

```diff
@@ -86,18 +86,19 @@ func (test *TestIterationRepository) TestCreateChildIteration() {
 	repo.Create(context.Background(), &i)
 
 	i2 := iteration.Iteration{
-		Name:     name2,
-		SpaceID:  uuid.NewV4(),
-		StartAt:  &start,
-		EndAt:    &end,
-		ParentID: i.ID,
+		Name:       name2,
+		SpaceID:    uuid.NewV4(),
+		StartAt:    &start,
+		EndAt:      &end,
+		ParentPath: iteration.ConvertToLtreeFormat(i.ID.String()),
 	}
 	repo.Create(context.Background(), &i2)
 
 	i2L, err := repo.Load(context.Background(), i2.ID)
 	require.Nil(t, err)
-	assert.NotEqual(t, uuid.Nil, i2.ParentID)
-	assert.Equal(t, i2.ParentID, i2L.ParentID)
+	assert.NotEmpty(t, i2.ParentPath)
+	expectedPath := iteration.ConvertToLtreeFormat(i.ID.String())
+	assert.Equal(t, expectedPath, i2L.ParentPath)
```

Comment:
correct, i have to use require.NotNil - done in https://github.com/almighty/almighty-core/pull/713/commits/496a66aefb24e66e665eb72d3b014098c7d76eb0#diff-ac105ac399c839c07e21bac7fbbb8651R103